remotes/docker/config/hosts.go

```diff
@@ -59,6 +61,7 @@ type HostOptions struct {
 	Credentials   func(host string) (string, string, error)
 	DefaultTLS    *tls.Config
 	DefaultScheme string
+	BaseHeader    http.Header
```

Comment:
Where is this used?

Reply:
Not used, trying to think if it should be used or not. The "base" headers should probably just be configured at the resolver level

Reply:
Removed, confirmed resolver option should solve the underlying intent here